src/components/tx/DecodedTx/index.tsx

```diff
@@ -59,7 +59,7 @@ const DecodedTx = ({
 }: DecodedTxProps): ReactElement | null => {
   const chainId = useChainId()
   const chain = useCurrentChain()
-  const isSwapOrder = isSwapConfirmationViewOrder(decodedData)
+  const isSwapOrder = isCoWConfirmationViewOrder(decodedData)
```

Comment:
Why rename this?

Reply:
because isCoWConfirmationViewOrder can contain both a swap and twap order.